reana_client/config.py

```diff
@@ -39,3 +39,14 @@
 
 WORKFLOW_ENGINES = ["serial", "cwl", "yadage"]
 """Supported workflow engines."""
+
+AVAILABLE_STATUS = [
```

Comment:
Minor detail: in this [pr](https://github.com/reanahub/reana-db/pull/90) we have also changed `WorkflowStatus` enum name to `RunStatus`. Maybe it's worth to name it here accordingly just for consistency?

Reply:
Going for RUN_STATUS , thanks 👍

Reply:
Done 👍